ballerina/tests/02-execute-basic-test.bal

```diff
@@ -129,8 +129,8 @@ isolated function testInsertAndSelectTableWithGeneratedKeys() returns sql:Error?
     string|int? insertedId = result.lastInsertId;
     if insertedId is string {
         sql:ParameterizedQuery query = `SELECT * from TestNumericTable where col_number = 31`;
-        stream<NumericRecord, sql:Error?> streamData = oracledbClient->query(query);
-        record {|NumericRecord value;|}? data = check streamData.next();
+        stream<record {}, sql:Error?> streamData = oracledbClient->query(query);
```

Comment:
Why was the typed stream changed to open record?

Reply:
The inserted row contains only one field (col_number). So when the `SELECT` query is executed, only one field in the record can be populated by non-null values. Hence a TypeCastError is returned. 

In earlier versions of SQL, in such situations, the whole record was not returned
https://github.com/ballerina-platform/ballerina-standard-library/issues/2985

Reply:
Ack